src/main/java/frc/robot/commands/Auto/ZoneShootAuto.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands.Auto;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.TestingDashboard;
+import frc.robot.subsystems.Auto;
+
+import frc.robot.commands.Drive.PIDDriveDistance;
+import frc.robot.commands.Drive.PIDTurnAngle;
+import frc.robot.commands.Shooter.PIDBottomShooter;
+
+public class ZoneShootAuto extends SequentialCommandGroup{
+    public ZoneShootAuto(String zone) {
+        
+            if (zone == "green") {
+                addCommands(
+                new PIDBottomShooter(0.3));
+            }
+            if (zone == "yellow") {
+                addCommands(
+                new PIDBottomShooter(0.4));
+            }
+            if (zone == "blue") {
+                addCommands(
+                new PIDBottomShooter(0.5));
+            }
+            if (zone == "red") {
+                addCommands(
+                new PIDBottomShooter(0.6));
+            }
+    }
+
+    public static void registerWithTestingDashboard() {
+        Auto auto = Auto.getInstance();
+        ZoneShootAuto cmd = new ZoneShootAuto("hello");
```

Comment:
Please change "hello" to one of the zone colors. Also, please create and register this command multiple times. (Once for each zone) Doing multiple registrations will allow us to select the correct zone from testing dashboard.

Reply:
Specificially, I'm looking for something like this:
        Auto auto = Auto.getInstance();
        ZoneShootAuto greenZone = new ZoneShootAuto("green");
        ZoneShootAuto yellowZone = new ZoneShootAuto("yellow");
        ZoneShootAuto blueZone = new ZoneShootAuto("blue");
        ZoneShootAuto redZone = new ZoneShootAuto("red");
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", greenZone);
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", yellowZone);
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", blueZone);
        TestingDashboard.getInstance().registerCommand(auto, "ZoneShootAuto", redZone);

Reply:
In the RobotContainer, should it still stay the same as the original? If so, then I made all of the changes necessary.

Reply:
Yes, the RobotContainer should stay the same.